otel/otel-using.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-temp"]
+= Using the {OTELShortName}
+include::_attributes/common-attributes.adoc[]
+:context: otel-temp
+
+toc::[]
+
```

Comment:
Assembly needs an intro (a single sentence would be okay)

Reply:
Added. Thank you.